src/renderer/components/Chart2/index.js

```diff
@@ -67,7 +67,15 @@ const ChartContainer: ThemedComponent<{}> = styled.div.attrs(({ height }) => ({
   position: relative;
 `;
 
-const Chart = ({ height, data, color, renderTickY, renderTooltip, valueKey = "value" }: Props) => {
+const Chart = ({
+  height,
+  data,
+  color,
+  renderTickY,
+  renderTooltip,
+  valueKey = "value",
+  dateFormat = "MMM D",
```

Comment:
Where is this used?

Reply:
Good question. I was wondering too. I saw it exists in Prop type but no component uses this prop. I'm just gonna delete it for now.